resources/plugins/test-editor-events/client/TestEditorEvents.js

```diff
@@ -111,3 +124,23 @@ function addModule(extensionModule) {
     };
   };
 }
+
+function addDmnModule(extensionModule) {
```

Comment:
Not a comment with regards to this PR, but yet another example that supports making this function part of the plugin helpers (https://github.com/camunda/camunda-modeler-plugin-helpers/issues/22).